parser/parser.y

```diff
@@ -6094,6 +6094,17 @@ StringType:
 		}
 		$$ = x
 	}
+|	"NATIONAL" "CHARACTER" FieldLen OptBinary OptCollate
```

Comment:
Can we define a new parser rule like:
```
CharOrCharacter:
     "CHAR" | "CHARACTER"
```
Then we could merge this into the first branch.

Reply:
@shenli 
NationalOpt could be empty,  "CHAR(10)" is legal, but "CHARACTER(10)" is illegal in MySQL. There must be a "NATIONAL" before "CHARACTER".

Reply:
OK